### PR TITLE
Optimization: use _ids and bulk_create in scorelog

### PR DESCRIPTION
**Pull request by phlax**
by using ids in scorelog the related models are not pulled from the db.

this PR also uses bulk_create for creating the scorelogs

### CODE REVIEW

File: pootle/apps/pootle_statistics/models.py [hunk 1]
```diff
@@ -320,10 +320,18 @@ def get_submission_info(self):
...
-                    ScoreLog.objects.create(**score)
+        if not self.needs_scorelog():
+            return
+
+        scorelogs_created = []
+        for score in ScoreLog.get_scorelogs(submission=self):
+            if 'action_code' in score and score['user'] is not None:
+                if not isinstance(score["user"], models.Model):
+                    score["user_id"] = score["user"]
+                    del score["user"]
```
Comment (unho) [inline pootle/apps/pootle_statistics/models.py]: This looks like handling a corrupt data situation. Shouldn't it be done in a data migration?

**Comment by ta2-1** [comment]
@phlax I tested scorelog.save() is triggered.
GTM after merge with further cleanup https://github.com/phlax/pootle/pull/1.

File: pootle/apps/pootle_statistics/models.py (hunk 1 above)
Comment (phlax) [inline pootle/apps/pootle_statistics/models.py]: @unho no its not that - sometimes user is a real user and sometime just a pk - this works out which and adjusts the dictionary used to create

**Comment by phlax** [comment]
closed in favour of https://github.com/translate/pootle/issues/5243

**Pull request closed without merging**